source/topologies/rs-to-sharded.txt

```diff
@@ -0,0 +1,115 @@
+sharded cluster using :program:`mongosync`. 
+
+.. note::
+
+   ``mongosync`` doesn't support sync from a sharded cluster to a replica set.
+
+To perform a sync between two replica sets, see :ref:`c2c-quickstart`.
+
+Before You Begin
+----------------
+
+Before you begin your sync:
+
+- Define your source replica set.
+- Define your destination sharded cluster.
+- Define an administrative user that has permissions in both clusters.
+
+You must also download ``mongosync``. To learn more, see 
+the :ref:`mongosync setup <c2c-quickstart-setup>` instructions.
+
+Steps 
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Connect the replica set and sharded cluster. 
+
+      ``mongosync`` must create an initial connection between the source 
+      replica set and the destination sharded cluster before you can start to 
+      sync data. 
+
+      To connect ``cluster0`` to ``cluster1`` with ``mongosync``, enter the
+      following command on one line:
+
+      .. include:: /includes/example-connection-string-general.rst
+
+      Follow the :ref:`connection instructions <c2c-conn-top-level>` 
+      for your cluster architecture to format your connection strings and 
+      connect to the :binary:`mongos` instance in your cluster. 
+
+   .. step:: Start synchronization.
+
+       Call the :ref:`start <c2c-api-start>` endpoint to initiate data 
+       synchronization. 
+       
+       To sync from a replica set to a sharded cluster, set the 
+       ``sharding`` option for the ``start`` command to shard collections on 
+       the destination cluster. For more information, see 
+       :ref:`Sharding Parameters <c2c-api-start-sharding>`.
+
+       Use the ``sharding.shardingEntries`` parameter to specify the collections 
+       to shard. Collections that you do not list in this array replicate as 
+       unsharded. For more information, see 
+       :ref:`Shard Replica Sets <c2c-shard-replica-sets>`.
+
+       The following example starts a sync from a replica set to a sharded
+       cluster: 
+
+       Request
+       ~~~~~~~
+
+       .. literalinclude:: /includes/api/requests/start-rs-shard.sh
+
+       Response
+       ~~~~~~~~
+
+       .. literalinclude:: /includes/api/responses/success.json
+          :language: json
+          :copyable: false
+
+Next Steps
+----------
+
+You can finalize a migration and transfer your application workload from the 
+source to the destination cluster using the ``mongosync`` 
+:ref:`cutover process <c2c-cutover-process>`.
+
+Limitations
+-----------
+
+- You cannot use the :ref:`reverse <c2c-api-reverse>` endpoint between a 
+  replica set and a sharded cluster. 
+
+.. include:: /includes/limitations-rs-to-sharded.rst
+
+- If the source replica set has :ref:`arbiters <c2c-faq-arbiters>`, the source 
+  replica set must have more than 2 non-arbiter nodes and you must sync from a 
+  non-arbiter node. 
+
+Learn More
+----------
+
+To learn more about ``mongosync``, see:
```

Comment:
Typically we just have bullet points for the Learn More section

```suggestion

```

Reply:
Deleting now!